README.md

```diff
@@ -35,6 +35,20 @@ and implementing query rewrite algorithms for data governance and query optimiza
 - Coral-Visualization [WIP]: Visualizes Coral SqlNode and RelNode trees and renders them to an output file.
 - Coral-Service: Service that exposes REST APIs that allow users to interact with Coral (see [Coral-as-a-Service](#Coral-as-a-Service) for more details).
 
+## Version Upgrades
+
+This project follows a versioning scheme in the format of x.y.z. As you integrate this project, it's essential to consider potential changes required when upgrading to different versions.
```

Comment:
Please do not direct the speech to someone by using "you" (or "we") etc. Try to phrase it differently.
It's --> It is.

Use "semantic versioning" term. x, y, z have names in that scheme.

Reply:
modified.